utils/ifupdown.sh.in

```diff
@@ -107,14 +107,14 @@ if [ "$MODE" = 'start' ]; then
   # See the code and comments in /sbin/bridge-stp for more details.
   "$mstpctl" addbridge "$IFACE"
 
-  if [ "$IF_MSTPCTL_MAXAGE" ]; then
-    "$mstpctl" setmaxage "$IFACE" "$IF_MSTPCTL_MAXAGE"
-  fi
-
   if [ "$IF_MSTPCTL_FDELAY" ]; then
     "$mstpctl" setfdelay "$IFACE" "$IF_MSTPCTL_FDELAY"
   fi
 
+  if [ "$IF_MSTPCTL_MAXAGE" ]; then
```

Comment:
one note it just occurred to me;
maybe it's worth adding a comment here describing the issue this solves;
maybe also in the commit;

over time, we tend to forget things; i usually search for things better in git history via CLI;

Reply:
How about what I mentioned in the bug report, what if someone now wants to decrease fdelay below 10 in the interface definition, and that fails? That would require setting fdelay again after maxage. What if we set fdelay twice: once before and once after maxage?

Reply:
Did you see my comment over on the issues thread?

https://github.com/mstpd/mstpd/issues/86#issuecomment-633889891

My shell scripting skills are limited, don't know how complicated that would be.

Reply:
Thanks, didn't see it. I think it's doable.

Reply:
I have yet to test it in my environment, but please check it when you have time.

Reply:
yes, this should run on a standard bourne shell (i.e. /bin/sh), but i'm a bit unclear on what that shell supports;
even busybox's shell (ash) seems to be able to handle $()
in this shell, there are already uses of $(), so you could use them;
if we get complaints to go for an older form, we might update to something like that;

regarding, `let`, i did run on some issues with it not working on bourne; so, using `expr` might be a good idea;

Reply:
I found that $(()) works (with help of shellcheck). This `should` now pass CI.